controllers/gatekeeper_controller.go

```diff
@@ -439,7 +445,11 @@ func (r *GatekeeperReconciler) validateWebhookDeployment() (error, bool) {
 	return nil, true
 }
 
-func getStaticAssets(gatekeeper *operatorv1alpha1.Gatekeeper) (deleteWebhookAssets, applyOrderedAssets, applyWebhookAssets, deleteCRDAssets []string) {
+func getStaticAssets(gatekeeper *operatorv1alpha1.Gatekeeper, isOpenshift bool) (deleteWebhookAssets, applyOrderedAssets, applyWebhookAssets, deleteCRDAssets []string) {
+	if !isOpenshift {
+		orderedStaticAssets = append(orderedStaticAssets, ServerCertFile)
```

Comment:
I think this would infinitely append `ServerCertFile` on every call.

Reply:
oh because of `orderedStaticAssets`

Reply:
Could you append it to `applyOrderedAssets` instead to avoid modifying `orderedStaticAssets` entirely?